src/bin/pg_autoctl/keeper_pg_init.c

```diff
@@ -278,6 +286,13 @@ keeper_pg_init_and_register_primary(Keeper *keeper)
 	KeeperConfig *config = &(keeper->config);
 	PostgresSetup *pgSetup = &(config->pgSetup);
 	char absolutePgdata[PATH_MAX];
+	char scrubbedConnectionString[MAXCONNINFO] = { 0 };
+	if (!parse_and_scrub_connection_string(config->monitor_pguri,
+										   scrubbedConnectionString))
+	{
+		log_error("Monitor connection string is unparseable");
```

Comment:
Another error message that would better read "Failed to parse the monitor connection string".